geometry_msgs/Pose pose
moveit_msgs/RobotState robot_state
moveit_msgs/MoveItErrorCodes solution_code